### DISCUSSION TITLE
[Feature] Show and group places differently when photos are taken from a plane

### DISCUSSION THREAD

**Discussion by teaalltr** [immich-app/immich#9682] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

So let's say you go for a plane trip and take some photos along the way, from the plane window. EXIF metadata are added to the image with the GPS coordinates and the altitude.
Using this information (specifically the altitude and possibly with some ML the kind of photo and subject too) we could group (and show on the map) the photos in a more meaningful way, let's say with a contiguous line between points on the map and have an album with the title "plane trip from here to there" or "plane trip ".
This would be pretty amazing to have, cause at the moment we have photos with random coordinates in the middle of the flight with the random corresponding places names down below the plane.

This could also be done for train trips for instance, but it's a bit more difficult I guess cause in that case we cannot rely on the altitude metadata.

It would also be awesome if we had some kind of API to gather all this random pieces of information about what we do and puzzle all them into a nice timeline like in Google Maps timeline feature, so this app and others can contribute to a local repository where some kind of other ML or rule-based service can process the infor in a cohesive and meaningful way (so toprovide the same level of event-aware assistance and reconstruction as Google does). Hope it makes sense

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
This is closely related enough to #6831 that I'll close it as a duplicate for now.
